plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesExt.kt

```diff
@@ -18,111 +18,175 @@ val MapPluginProviderDelegate.gestures: GesturesPlugin
 
 /**
  * Add a callback that is invoked when the map is clicked.
+ * Gesture plugin with id = [Plugin.MAPBOX_GESTURES_PLUGIN_ID] must be added while constructing
+ * [com.mapbox.maps.MapView] as part of [com.mapbox.maps.MapInitOptions.plugins].
+ *
+ * @see [com.mapbox.maps.MapInitOptions]
```

Comment:
one thing that also came to my mind - specify `@throws` here telling what exception will be thrown if gesture plugin is not added

Reply:
Done